scrapers/oldje.yml

```diff
@@ -0,0 +1,45 @@
+# This scraper does not parse older scenes which use a different page layout
+name: Oldje
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - oldje.com/video
+    scraper: oldjeScraper
+  - action: scrapeXPath
+    url:
+      - oldje-3some.com/videos
+    scraper: oldje3someScraper
+xPathScrapers:
+  oldjeScraper:
+    scene:
+      Studio:
+        Name: Oldje
+      Title:
+        selector: //h1[@class="preview_header"]/span[1]
```

Comment:
this doesnt seem to work with a few scenes i tested
```
https://www.oldje.com/video/arteya-and-alessandra-jane-and-patricia-sun-and-baby-jewel-and-lady-dee/memories-of-an-oldje
https://www.oldje.com/video/keira-flow/vlogging-and-boning
https://www.oldje.com/video/kittina-ivory/naughty-cleaning-lady
```
while `//h1[@class="inlay_title"]` seems to work
Do you have same sample URLs where this worked?